src/vivarium_census_prl_synth_pop/utilities.py

```diff
@@ -532,3 +532,10 @@ def copy_from_household_member(
             )
 
     return pop
+
+
+def write_metadata_file(final_output_dir: Path, label_version: str) -> None:
+    data_version = pd.Series(label_version)
+    metadata = pd.DataFrame({"data_version": data_version})
+    outpath = final_output_dir / "metadata.parquet"
+    metadata.to_parquet(outpath)
```

Comment:
Do we want the metadata file to be parquet or somethign else like json?

Reply:
Same response as above - It can be whatever we want it to be. I was trying to align with our other outputs from post-processing that are either hdf or parquet. We don't know all of what will be going into this file and what the format should be so I was just trying to stay consistent.